src/styles/stop-watch.scss

```diff
@@ -0,0 +1,62 @@
+.stop-watch {
+  &__container {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```

Comment:
For the container class, add a width, for example 500px, so that the clock does not stop when the cursor is on the page, only when you point to the clock itself. Also, don't forget the stop-watch class to align the block to the center of the page using flexes.